RELEASING.md

```diff
@@ -0,0 +1,35 @@
+# Bookkeeper release guide
+
+## Docker
```

Comment:
Maybe we can move this doc under the docker directory?

Reply:
ok